src/k8s-engine/lima.ts

```diff
@@ -155,7 +156,9 @@ const ALPINE_VERSION = '3.14.3';
  */
 const VDE_DIR = '/opt/rancher-desktop';
 const RUN_LIMA_LOCATION = '/private/var/run/rancher-desktop-lima';
-const LIMA_SUDOERS_LOCATION = '/private/etc/sudoers.d/rancher-desktop-lima';
+const LIMA_SUDOERS_LOCATION = '/private/etc/sudoers.d/zzzzz-rancher-desktop-lima';
+// Issue 1444, moving to version 1.0.1
+const PREVIOUS_LIMA_SUDOERS_BASENAME = '/private/etc/sudoers.d/rancher-desktop-lima';
```

Comment:
I'd have expected `basename` to just be `rancher-desktop-lima` (without the containing directory)?

Reply:
Yeah, the code changed but left the old name behind.